tests/test_markup.py

```diff
@@ -1,6 +1,5 @@
-import os
 import re
-import tempfile
+import pytest
```

Comment:
Please add a blank line above pytest. It's common to separate the stdlib imports from installed packages.